manifests/init.pp

```diff
@@ -366,6 +368,7 @@
   Array[String] $dhcp_option_domain = $::foreman_proxy::params::dhcp_option_domain,
   Optional[Array[String]] $dhcp_search_domains = $::foreman_proxy::params::dhcp_search_domains,
   String $dhcp_interface = $::foreman_proxy::params::dhcp_interface,
+  Optional[Variant[Array[String], String]] $dhcp_additional_interfaces = $::foreman_proxy::params::dhcp_additional_interfaces,
```

Comment:
I'd prefer if this was simply ```Array[String]``` which defaults to ```[]``` in ```params.pp``` since it removes the need to clean it up.

Reply:
Ok, will do.